internal/cli/cmd/kubeblocks/upgrade.go

```diff
@@ -116,6 +118,15 @@ func (o *InstallOptions) Upgrade() error {
 		return err
 	}
 
+	// double check for KubeBlocks upgrade
+	if !o.autoApprove {
+		upgradeWarn := printer.BoldYellow(fmt.Sprintf("Warning: You're attempting to change KubeBlocks version from %s to %s, potentially impacting cluster functionality, particularly with downgrades. \nEnsure you proceed after reviewing detailed release notes at https://github.com/apecloud/kubeblocks/releases.", kbVersion, o.Version))
```

Comment:
For upgrades, upgrading to a higher version is a more common and recommended operation, in which case only a simple secondary confirmation message needs to be output; for downgrades to lower versions, it is something we do not recommend, and more detailed risk descriptions and suggestions should be provided.

Reply:
> 

Get it! I will implement a condition to differentiate whether the user is dong an upgrade or a downgrade